test/replication/qsync_basic.result

```diff
@@ -589,6 +616,9 @@ test_run:switch('replica')
  | ---
  | - true
  | ...
+test_run:wait_lsn('replica', 'default')
```

Comment:
No need to do `wait_lsn` here. You wait until the transaction is committed on master, so the value must be visible on replica too (unless `memtx_mvcc` gets turned on by default in tests)

Reply:
right